docker-compose/federation/README.md

```diff
@@ -266,7 +261,7 @@ Similarly, once this registration entry is created, when the quotes service asks
 
 That is about it. Now all the pieces are in place to make federation work and demonstrate how the webapp is able to communicate with the quotes service despite having identities with different trust domains.
 
-# Federation Example Using SPIFFE Authentication with SPIRE 0.11.0
+# Federation Example Using SPIFFE Authentication with SPIRE 1.0.0
```

Comment:
I'm wondering if we should include the SPIRE version in this title or move it to the _Prerequisites_ section.
Something Like:
```
Prerequisites

Two SPIRE Server instances running version 1.0.0
Two SPIRE Agents running version 1.0.0. One connected to one SPIRE Server, and the second connected to the other SPIRE Server
Two workloads that needs to communicate each other via mTLS, and use the Workload API to get SVIDs and trust bundles
```
What do you think?